sdk/storage/azure-storage-blob-batch/CHANGELOG.md

```diff
@@ -1,12 +1,7 @@
 # Release History
 
-## 12.19.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 12.19.0 (2023-07-11)
+- Added support for 2023-01-03 service version.
```

Comment:
We should put these service version updates under features added for all packages.

```suggestion
### Features Added
- Added support for the `2023-01-03` service version.
```